### ISSUE TITLE
Team created by new member

### ISSUE THREAD

**Issue by jancborchardt** [opensourcedesign/organization#104]
Hey @R2in, you just joined our organization, created this team https://github.com/orgs/opensourcedesign/teams/twcotw, and requested the core team to be parent of that. What's up with that? :)

cc @opensourcedesign/core

**Comment by m1guelpf** [comment]
According to a quick Google search, TWCOTW seems to mean "The Worst Collection On The World"

**Comment by jancborchardt** [comment]
@m1guelpf looking at @R2in's profile it seems to mean "Thoso Without Courage, Oppress The Weak."

@simonv3 the audit log says you invited @R2in, is that correct? Seems like it might have been @m1guelpf's auto-adding tool? If we have people joining who modify things like add groups and start projects with cryptic unrelated names, we might have to get rid of that automatic mechanic … :/

**Comment by simonv3** [comment]
Yeah, is there a way to make sure that they can't just create repositories when being a member?

**Comment by ei8fdb** [comment]
Something tells me it's a bot. If I am wrong I'll apologise to @R2in, in person.

**Comment by jancborchardt** [comment]
@simonv3 just limited that to owners. It's a bit of a bummer we have to do that, but better than losing stuff. :/
Creating (and maybe editing/deleting) teams doesn't seem to be possible to limit? Will contact Github support.

**Comment by ei8fdb** [comment]
@m1guelpf In which case I apologise to you @R2in.

**Comment by jancborchardt** [comment]
For the record, I now removed @R2in from the organization. Reasons:
- No feedback or communication from them at all in this thread or anywhere else.
- Merged a pull request (https://github.com/opensourcedesign/opensourcedesign.github.io/pull/169) without any comment. It was non-disruptive, but still it’s not how we work in the open.
- Creating strange teams with names as if spraying graffiti tags, as mentioned above.
- The Github profile https://github.com/R2in which belongs to the person is not accessible/visible (anymore – it was previously). Maybe a @github glitch, but seeing from the real name (consisting of a bunch of special characters) of the user it might also be intentional.

Am happy to add them again if they properly communicate. :)
